source/game/kart/KartObjectManager.cc

```diff
@@ -5,7 +5,12 @@
 
 namespace Kart {
 
-void KartObjectManager::init() {}
+void KartObjectManager::init() {
+    for (size_t i = 0; i < m_count; ++i) {
+        m_objects[i]->initImpl();
+        m_objects[i]->prepare();
```

Comment:
Could we explore using a `std::span` here? I just know you brought this up earlier. Maybe it's applicable here, so that we could `for (auto& elem : m_objects)`

Reply:
I would like to experiment with `std::span` after merge.